eng/Versions.props

```diff
@@ -38,6 +38,7 @@
     <!-- Dependencies from https://github.com/aspnet/AspNetCore -->
     <MicrosoftAspNetCoreAppRuntimewinx64PackageVersion>3.1.3</MicrosoftAspNetCoreAppRuntimewinx64PackageVersion>
     <MicrosoftAspNetCoreAppRefPackageVersion>3.1.3</MicrosoftAspNetCoreAppRefPackageVersion>
+    <MicrosoftAspNetCoreAppRefInternalPackageVersion>3.1.3</MicrosoftAspNetCoreAppRefInternalPackageVersion>
```

Comment:
As a follow-up to this, I'm going to produce a `Microsoft.AspNetCore.App.Ref` package to identify the proper non-stable ref pack version, for the targeting-pack .msi. For now I'm just creating the property here pre-emptively, will link the issue once I create it